crates/ere-risczero/build_script_template.rs

```diff
@@ -1,62 +0,0 @@
-// This is ere-risczero/build_script_template.rs
```

Comment:
Just trying to figure out why this is not needed anymore

Reply:
Preivously we use the `build.rs` to call the [`embed_methods`](https://github.com/risc0/risc0/blob/main/risc0/build/src/lib.rs#L844-L846), but it seems easier to just call the `cargo risczero build` which also [outputs the elf path and image id](https://github.com/risc0/risc0/blob/main/risc0/cargo-risczero/src/commands/build.rs#L58), so we could remove this.